src/containers/CategoryFilter.js

```diff
@@ -0,0 +1,41 @@
+import PropTypes from 'prop-types';
+import { useState } from 'react';
+
+const categories = [
+  'All',
+  'Action',
+  'Biography',
+  'History',
+  'Horror',
+  'Kids',
+  'Learning',
+  'Sci-Fi',
+];
+
+const CategoryFilter = ({ filterChange }) => {
+  const [category, setCategory] = useState('All');
+  const handleCategoryChange = (e) => {
+    filterChange(e.target.value);
+    setCategory(e.target.value);
+  };
```

Comment:
- [ ] Please move this component to the `src/components` folder as it is a presentational component placed in `src/components` directory [screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone4/presentationalCategoryFilter.png)

Reply:
Thanks, @mikenath223 for your feedback; I just addressed the issue.

Kindly request a pull request before I merge the feature branch to the develop branch.
Regarss,
Temesghen T Bahta